####################################################################################################
#
#    Copyright (c) 2023  ___     ___
#                       \\  \\  /  /
#                        \\  \\/  /
#                         /  /\\  \\
#                        /__/  \\__\\  Fixposition AG
#
####################################################################################################
#
# Fixposition Covariance Warning Message
#
#
####################################################################################################

Header header
geometry_msgs/Vector3 jump         # Position change
geometry_msgs/Vector3 covariance   # Covariance
